ios/FluentUI/Drawer/DrawerTokenSet.swift

```diff
@@ -5,31 +5,31 @@
 
 import UIKit
 
-/// Design token set for the `Drawer` control
-public class DrawerTokenSet: ControlTokenSet<DrawerTokenSet.Tokens> {
-    public enum Tokens: TokenSetKey {
-        /// Corner radius for the popover style `Drawer` control.
-        case cornerRadius
+public enum DrawerToken: Int, TokenSetKey {
```

Comment:
Not important at all but I'm curious about why we're opting for singular token naming as opposed to `DrawerTokens`

Reply:
Good question! If you look at [Apple's docs on enumerations](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/enumerations), enum types should generally be singular. Example from their docs:
```swift
enum Planet {
    case mercury, venus, earth, mars, jupiter, saturn, uranus, neptune
}

let somePlanet = Planet.earth
switch somePlanet {
case .earth:
    print("Mostly harmless")
default:
    print("Not a safe place for humans")
}
// Prints "Mostly harmless"
```

We've been slowly moving our enums from plural to singular across the library (see Global and Alias tokens, which have already switched), but it's definitely a work in progress.